source/deployment/elasticsearch.rst

```diff
@@ -49,9 +49,9 @@ Follow these steps to connect your Elasticsearch server to Mattermost and genera
     - Note: For AWS Elasticsearch leave this field blank.
   d) Set **Enable Cluster Sniffing** (Optional). Sniffing finds and connects to all data nodes in your cluster automatically.
     - Note: For AWS Elasticsearch this field should be set to ``false``.
-4. Click **Test Connection** and **Save** the configuration.
+4. Select **Test Connection** and **Save** the configuration.
   - If the server connection is unsuccessful you will not be able to save the configuration or enable searching with Elasticsearch.
-5. Build the post index of existing posts by clicking **Build Index**.
+5. Build the post index of existing posts by selecting **Build Index**.
   - This process can take up to a few hours depending on the size of the post database and number of messages. The progress percentage can be seen as the index is created. To avoid downtime set **Enable Elasticsearch for search queries** to ``false`` so that database search is available during the indexing process.
 6. Enable Elasticsearch by setting **Enable Elasticsearch for search queries** to ``true``.
   - Note: It is recommended that bulk indexing be completed before enabling Elasticsearch, otherwise search results will be incomplete. When this setting is ``false``, database search is used for all search queries.
```

Comment:
```suggestion
  - **Note:** It's recommended that bulk indexing be completed before enabling Elasticsearch, otherwise search results will be incomplete. When this setting is ``false``, database search is used for all search queries.
```